aws/src/integration/java/org/apache/iceberg/aws/lakeformation/LakeFormationTestBase.java

```diff
@@ -351,7 +351,7 @@ private static void registerResource(String s3Location) {
       // when a resource is registered, LF will update SLR with necessary permissions which has a propagation delay
       waitForIamConsistency();
     } catch (AlreadyExistsException e) {
-      LOG.warn("Resource {} already registered. Error: {}", arn, e);
+      LOG.warn("Resource {} already registered. Error: {}", arn, e.getMessage(), e);
```

Comment:
this was actually producing an errorprone error (same for the other places around logging)

Reply:
This is just an integration test, so not a big deal. But does it make sense to log both `e.getMessage()` along with `e`?

I don't want to change the aws integration tests too much, but if we weren't inside the aws integration tests I would suggest `LOG.warn("Resource {} already registered", arn, e)` instead.